### PR TITLE
Doc update for SDK 6 0.20.0

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**
https://docs.microej.com/en/feature-sdk6_develop

### CODE REVIEW

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 1]
```diff
@@ -3,6 +3,46 @@
...
+
+[0.20.0] - 2024-10-23
+~~~~~~~~~~~~~~~~~~~~~
+
+Added
+"""""
+
+- Allow to build a Feature from a WPK and a Kernel.
+- Allow to filter the BSP files included in a published VEE Port.
+- Check that the MicroEJ Runtime Library is not in the BSP before publishing a VEE Port.
```
Comment (denisbedin) [inline SDK6UserGuide/sdk6ChangeLog.rst:16]: The support of VEE Port as included build is not in the changelog.

File: SDK6UserGuide/sdk6MigrationNotes.rst [hunk 2]
```diff
@@ -8,6 +8,48 @@ Migration Notes
...
+The following plugins and configurations have been renamed:
+
+- plugin ``com.microej.gradle.j2se-library`` to ``com.microej.gradle.jse-library``.
+- plugin ``com.microej.gradle.runtime-api`` to ``com.microej.gradle.runtime-environment``.
+- configuration ``microejRuntimeApi`` to ``microejRuntimeEnvironment``.
+
+The previous plugins and configurations are still supported but are deprecated and will be removed in the next major version.
+It is recommended to update your project now.
+
+Virtual Device Build
```
Comment (denisbedin) [inline SDK6UserGuide/sdk6MigrationNotes.rst:42]: The VD must be removed from the default published artifacts in the [Publish a Project](https://docs.microej.com/en/feature-sdk6_develop/SDK6UserGuide/publishProject.html) page and [this section](https://docs.microej.com/en/feature-sdk6_develop/SDK6UserGuide/buildVirtualDevice.html#skip-virtual-device-build-by-default) must be updated.

File: SDKUserGuide/licenses.rst [hunk 3]
```diff
@@ -6,7 +6,7 @@ Licenses
 SDK EULA
 --------
 
-MICROEJ SDK is licensed under the `SDK End User License Agreement (EULA) <https://repository.microej.com/licenses/sdk/LAW-0011-LCS-MicroEJ_SDK-EULA-v3.1B.txt>`_, which covers the following elements:
+MICROEJ SDK is licensed under the `SDK End User License Agreement (EULA) <https://repository.microej.com/licenses/sdk/LAW-0011-LCS-MicroEJ_SDK-EULA-v3.1C.txt>`_, which covers the following elements:
```
Comment (denisbedin) [inline SDKUserGuide/licenses.rst:9]: When I click on the link, it opens the License 3.1B : https://docs.microej.com/en/feature-sdk6_develop/SDK6UserGuide/licenses.html.
The `accept-microej-sdk-eula-v3-1b` property has not been updated.

**Review by denisbedin — CHANGES_REQUESTED** [review]
The "Create a project" page must also be updated with the Java SE Library and Runtime Environment templates since they are now available in IDE plugins and on github. Some of the  IDE screenshots must be changed since the wizards have been modified. I will do it in a separate branch.

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/sdk6ChangeLog.rst:16]: Done.

File: SDK6UserGuide/sdk6MigrationNotes.rst (hunk 2 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/sdk6MigrationNotes.rst:42]: Done.

File: SDKUserGuide/licenses.rst (hunk 3 above)
Comment (tdelhomenie-microej) [reply SDKUserGuide/licenses.rst:9]: Done.

Comment (denisbedin) [reply SDKUserGuide/licenses.rst:9]: Actually you didn't update the right page, this one is the License page for SDK 5, not 6.

**Review by denisbedin — CHANGES_REQUESTED** [review]

File: SDKUserGuide/licenses.rst (hunk 3 above)
Comment (tdelhomenie-microej) [reply SDKUserGuide/licenses.rst:9]: Actually I did bu did not push...

Comment (denisbedin) [reply SDKUserGuide/licenses.rst:9]: ok, but this Licence page must not be updated since we didn't added the 3.1C to SDK 5.

Comment (tdelhomenie-microej) [reply SDKUserGuide/licenses.rst:9]: Not sure, it would be weird to not refer to the latest EULA in the documentation.

Comment (denisbedin) [reply SDKUserGuide/licenses.rst:9]: ok

**Review by denisbedin — APPROVED** [review]

**Pull request merged**